source/MRViewer/MRViewport.h

```diff
@@ -145,6 +145,25 @@ class Viewport
          bool allowAlphaSort = false ///< If not null and the object is semitransparent, enable alpha-sorting.
     ) const;
 
+    // ...
+    using PickRenderObjectPredicate = std::function<bool ( const VisualObject*, ViewportMask )>;
+    // ...
+    struct PickRenderObjectParams
+    {
+        // ...
+        PickRenderObjectPredicate predicate;
+        // ...
+        uint16_t pickRadius = 0;
+        // ...
+        bool exactPickFirst = true;
+
+        static PickRenderObjectParams defaults()
```

Comment:
constexpr?

Reply:
There's no constexpr version of std::function.